### PR TITLE
Please My Mobile Portfolio Middle Section

### PR DESCRIPTION
**Pull request by efoosag**
Thank you for you review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,6 +42,258 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="nav-link">
+            <ul>
+              <li><a href="#">html</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">javascript</a></li>
```
Comment (AnselemOdims) [inline index.html:70]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are not links, so please remove the anchor `a` tag.

File: index.html [hunk 2]
```diff
@@ -42,6 +42,258 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+            </ul>
+          </div>
+          <div class="see-project">
+            <p><a href="#">See Project</a></p>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card1">
```
Comment (AnselemOdims) [inline index.html:78]: - [ ] Please make sure you your design sticks to the original [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design guidelines(colors, fonts, margins, padding, width etc)
- [ ] Your button needs to match the original design. Please kindly adjust.

### Your Design
image

### Original Design
image

File: index.html [hunk 3]
```diff
@@ -42,6 +42,258 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+          </div>
+          <div class="see-project">
+            <p><a href="#">See Project</a></p>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <h2>About My Self</h2>
+        <div class="about-content">
+          <p>
```
Comment (AnselemOdims) [inline index.html:174]: - [ ] Please make sure you your design sticks to the original [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design guidelines(colors, fonts, margins, padding, width etc)

### Your Design
image

# Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @efoosag , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly make your PR title and summary more descriptive about the recent changes you made.
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!. Kindly tag me using @AnselemOdims for a quicker response.

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (efoosag) [reply index.html:70]: Done

File: index.html [hunk 4]
```diff
@@ -42,6 +42,258 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="connect">LET’S CONNECT</div>
```
[the remark this answers is not in the document]
Comment (efoosag) [reply index.html:181]: Done

File: index.html (hunk 2 above)
Comment (efoosag) [reply index.html:78]: Done

File: index.html (hunk 3 above)
Comment (efoosag) [reply index.html:174]: Done

File: index.html [hunk 5]
```diff
@@ -23,25 +23,269 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/medium.svg" alt="Medium" /></a>
+            </li>
+          </ul>
+        </nav>
+        <div class="get-my-resume">
+          <p class="button"><a href="#">Get My Resume</a></p>
+        </div>
```
Comment (BenMukebo) [inline index.html:205]: - [ Optional] Please implement  on the `get my resume and see project` button these actions as defined in this image  

acti

**Review by BenMukebo — APPROVED** [review]
### *** Hi @efoosag, *** 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 


Cheers and Happy coding!👏👏👏

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**